### ISSUE TITLE
cant get the time-min-milliseconds rule to work

### ISSUE THREAD

**Issue by hendrikeng** [stylelint/stylelint#2323]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

cant get the rule 
 "time-min-milliseconds": [
      "init",
      {
          "100"
      }
    ],
to work , i tried to pass it with init as option, as a string ..it always says init is not an option

> What stylelint configuration is needed to reproduce this issue?

```json
"time-min-milliseconds": [
      "init",
      {
          "100"
      }
    ],
```

> Which version of stylelint are you using?

 `7.7.1`

**Comment by m-allanson** [comment]
time-min-milliseconds was released in stylelint version 7.8.0. Can you try upgrading your version of stylelint?

**Comment by jeddy3** [comment]
Also, the rule excepts an `int` as its primary option. So you'll need to configure it as so:

```js
"time-min-milliseconds": 100
```

Out of curiosity, where did you do get the following construct from?

```
"time-min-milliseconds": [
      "init",
      {
          "100"
      }
    ],
```

If it's somewhere in the docs it needs to be fixed.

**Comment by hendrikeng** [comment]
my bad, it works now with:
```"time-min-milliseconds": 100```
i tried it before but since i forgot to update to 7.8.0 i came up with the weird construct...its not in the docs..thanks a lot guys

**Comment by jeddy3** [comment]
No worries. We're glad you got it working :)
